src/Debug.js

```diff
@@ -63,24 +63,25 @@ function indentChildren(childrenStrs, indentLength) {
     : '';
 }
 
-export function debugNode(node, indentLength = 2) {
+export function debugNode(node, indentLength = 2, options = {}) {
   if (typeof node === 'string' || typeof node === 'number') return escape(node);
   if (!node) return '';
 
-  const childrenStrs = compact(childrenOfNode(node).map(n => debugNode(n, indentLength)));
+  const childrenStrs = compact(childrenOfNode(node).map(n => debugNode(n, indentLength, options)));
   const type = typeName(node);
-  const props = propsString(node);
-  const beforeProps = props ? ' ' : '';
-  const nodeClose = childrenStrs.length ? `</${type}>` : '/>';
+
+  const props = options.ignoreProps ? '' : propsString(node);
+  const beforeProps = (options.ignoreProps || !props) ? '' : ' ';
```

Comment:
if the `ignoreProps` option is set, then `props` will be the empty string, which is falsy - i think you can just revert this line.